Bpi/Sdk/Bpi.php

```diff
@@ -94,7 +94,7 @@ public function push(array $data)
         $nodes->firstItem('name', 'node')
             ->template('push')
             ->eachField(function ($field) use ($data) {
-                if (!empty($data[(string)$field]))
+                if (!isset($data[(string)$field])) // nb: check on empty() produce errors.
```

Comment:
In general I would prefer a comment on *why* we use `isset` and not `empty`. I am guessing that these templates support empty values but require callers to define them if so.